biz.aQute.remote/bnd.bnd

```diff
@@ -8,12 +8,16 @@ aQute.agent.server.port = 29998
 
 -buildpath: \
 	osgi.core;version=latest;maven-scope=provided,\
+    biz.aQute.bndlib;version=latest,\
```

Comment:
Is it wise to move bndlib ahead of aQute.libg? This would use the libg from bndlib instead of directly from the project. All the other projects put libg before bndlib.